### ISSUE TITLE
Nested enumerated lists end up on the same line

### ISSUE THREAD

**Issue by msrd0** [msrd0/cargo-doc2readme#138]
```rust
//! Several items:
//! 1. Item one
//! 2. Item two
//!     1. Subitem one
//!     2. Subitem two
```

produces the following incorrect result:

```markdown
Several items:

 1. Item one
 1. Item two 1. Subitem one
	 1. Subitem two
```

This was reported by @niluxv in #137

**Comment by niluxv** [comment]
Well, there is now still a bug:

```rust
//! List:
//! 1. Item one
//!    * Item
//!    * Item
//! 2. Item two
```

gives

```md
 1. Item one
	 - Item
	 - Item
	
 - Item two
```

This is fixed by the pop I commented on in #140

**Comment by msrd0** [comment]
Oh yeah that's another bug. Guess I've never used this tool with nested lists before ...
